index.qmd

```diff
@@ -9,125 +9,146 @@ execute:
   warning: false
 ---
 
-## What Does the NHP Model Do?
+## What is it for?
 
-The NHP demand model predicts _future hospital activity_, helping NHS trusts understand how many patients might visit their hospitals (admissions) and how long they might stay there (beddays).
+The New Hospital Programme (NHP) is building hospitals across England. To help plan for the future, [The Strategy Unit](https://www.strategyunitwm.nhs.uk/) developed a tool to predict how many people might use these hospitals and how long they’ll need care. 
 
-::: {.callout-note}
-The model predictors activity for inpatients, outpatients and A&E attendances.
-:::
+## What does it do?
 
-The model helps NHS trust:
+The model estimates:
 
-- Plan for future hospital activity with with bespoke data-driven forecasts
-- Explore a range of possible outcomes, helping prepare for uncertainty
-- Breakdown forecasts by site, activity type, age group, and specialty
+- How many people will need treatment (admissions).
+- How long patients might stay in the hospital (bed days)
 
+It looks at all types of hospital visits, like surgeries, check-ups, and emergencies, and breaks it down by things like age, sex, and treatment.
 
 ```{r multiple}
 value_boxes <- list(
   bslib::value_box(
-    title = "Number of settings",
-    value = "95",
-    showcase = bsicons::bs_icon("gear"),
+    title = "NHS trusts using the model",
+    value = "27+",
+    showcase = bsicons::bs_icon("hospital"),
     theme = "primary"
   ),
   bslib::value_box(
-    title = "Years into the future",
-    value = "20",
+    title = "Looking ahead",
+    value = "2041/42",
     showcase = bsicons::bs_icon("graph-up"),
     theme = "primary"
   ),
   bslib::value_box(
-    title = "NHS trusts using it",
-    value = "20",
-    showcase = bsicons::bs_icon("hospital"),
+    title = "Customisation options",
+    value = "90+ ",
+    showcase = bsicons::bs_icon("gear"),
     theme = "primary"
   ),
   bslib::value_box(
-    title = "Rows of data",
-    value = "30 million",
+    title = "Rows of real hospital data",
+    value = "140 million",
     showcase = bsicons::bs_icon("journal-medical"),
-    theme = "primary",
+    theme = "primary"
   )
 )
 
+
 bslib::layout_column_wrap(
   width = "150px",
-  !!!value_boxes
+  !!!value_boxes[1:2]
 )
+
 ```
 
-## How Does the NHP Model Work?
+## What factors does it consider?
 
-### How it runs
+The tool considers many things that can change how much people need hospitals in the future:
 
-1. **Uses real data**: Based on 2019/20 hospital activity (HES data)  
-1. **Growth**: More people, living longer means more activity
-1. **User changes**: Enter expected changes for hospital activity, given as ranges
-1. **Simulations**: Runs hundreds of scenarios to explore possible futures 
+### Population changes
```

Comment:
I think given that you have "non demographic growth" it makes more sense to call this "demographic growth" because they are kind of kin